### PR TITLE
chore: cleanup header, Navbar

### PR DESCRIPTION
**Pull request by eragon512**
**What changed? Why?**
Follow-up on comments from PR #329 

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: template/src/components/header/AccountInfoPanel.tsx [hunk 1]
```diff
@@ -16,20 +16,14 @@ export function AccountInfoPanel() {
...
-          </div>
+          <div className="font-inter w-32 text-base font-medium text-white">{ensName}</div>
+          <OnchainAddress
+            address={address}
+            className="font-inter w-32 text-sm font-medium text-zinc-400"
+          />
         </div>
       </div>
-      <div className="h-px self-stretch bg-zinc-400 bg-opacity-20" />
+      <hr className="h-px self-stretch bg-zinc-400 bg-opacity-20" />
```
Comment (arsood) [inline template/src/components/header/AccountInfoPanel.tsx:26]: Why the change from div to hr? Would prefer a div over an hr.

Comment (eragon512) [reply template/src/components/header/AccountInfoPanel.tsx:26]: In this case, this element is supposed to render a separator between the onchain address and the logout button

Having this be an hr element instead of a seems to fit better from a Semantic HTML POV, and the feedback from the earlier PR was to use more Semantic HTML

Comment (eragon512) [reply template/src/components/header/AccountInfoPanel.tsx:26]: Link to the feedback: https://github.com/coinbase/build-onchain-apps/pull/329#discussion_r1481025596

**Comment by arsood** [comment]
@eragon512 let's run prettier to get the build passing and then we can approve and merge.

**Comment by eragon512** [comment]
@arsood updated :+1:

**Review by arsood — APPROVED** [review]
